binary_search_tree/BinarySearchTree.py

```diff
@@ -0,0 +1,182 @@
+# Node for Binary Search Tree
+class Node:
+    def __init__(self, data):
+        self.data = data
+        self.left = None
+        self.right = None
+
+
+# Function for find the left side right most node in binary search tree
+def find_leftside_rightmost(node):
+    left_side_rightmost = node.left
+    prev_node = node
+    while left_side_rightmost.right is not None:
+        prev_node = left_side_rightmost
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpz8i066jt/binary_search_tree/BinarySearchTree.py
+++ b/tmp/tmpz8i066jt/binary_search_tree/BinarySearchTree.py
@@ -16,6 +16,8 @@
     return left_side_rightmost, prev_node
 
 # Binary Search Tree declaration
+
+
 class BST:
     def __init__(self):
 
```